modelzoo/WnD/TensorFlow2/wnd.patch

```diff
@@ -21621,42 +21621,42 @@ index d673f74..78d48ed 100644
 --- a/modelzoo/WnD/TensorFlow2/main.py
 +++ b/modelzoo/WnD/TensorFlow2/main.py
 @@ -1,5 +1,5 @@
- # Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
--#
-+# Modifications copyright Intel
- # Licensed under the Apache License, Version 2.0 (the "License");
- # you may not use this file except in compliance with the License.
- # You may obtain a copy of the License at

```

Comment:
was this the original copy right? 
if so, should modifications be made on top of it? rather than below it?

Reply:
the original copyright is "Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved." and we added "Modifications copyright Intel" since we made optimizations based on NV's code.